crates/planus-codegen/src/templates/rust/enum.template

```diff
@@ -43,7 +43,9 @@ impl ::core::convert::From<{{info.name}}> for {{info.repr_type}} {
     }
 }
 
-impl ::planus::Primitive for {{ info.name }} {
+// # Safety
```

Comment:
I'm a bit surprised to see markdown in a normal comment. Usually I only see markdown doc-comments. But perhaps that's just how safety comments are usually written?

Reply:
That's fair. If you are intrigued by that format because the lint suggested it: https://rust-lang.github.io/rust-clippy/master/index.html#missing_safety_doc

Reply:
Those are docstrings though with a triple `/`, so if you want to follow the style in your own link, then you should probably change it.

I can't say that I feel strongly about this though, so if you prefer to keep them as comments instead of docstrings then that's fine with me as well.

Reply:
Ok I changed then to triple `/`. I think that is it.